extensions/cornerstone/src/tools/ImageOverlayViewerTool.tsx

```diff
@@ -46,6 +47,15 @@ class ImageOverlayViewerTool extends AnnotationDisplayTool {
     this._cachedOverlayMetadata = new Map();
   };
 
+  protected getReferencedImageId(viewport: IStackViewport | IVolumeViewport): string {
```

Comment:
Why can't we use the one from AnnotationDisplayTool?

Reply:
Since that one requires worldPos and other coordinates, which we don't have here